### PR TITLE
Move all document.ready functions to just one file per page #7002

### LINKED ISSUE #7002: Move all document.ready functions to just one file per page
For example, the `$(document).ready` for instructor courses page should only come from `instructorCourses.js` (or similar name) and all other JS files used by that page should not have its own `$(document).ready` functions.

### PR DESCRIPTION
**Pull request by wkurniawan07**
Fixes #7002 

**Outline of Solution**

- Made sure that there is one `main` script per page, named similarly with the page in question
- There were some scripts used only in one page; these were merged with the `main` scripts
- All `$(document).ready` calls in non-`main` scripts were moved to the main counterparts

### CODE REVIEW

**Comment by thenaesh** [comment]
Wow, another one of these monstrosities. I'll review tomorrow morning. 😝

File: src/test/java/teammates/test/cases/browsertests/AllJsTests.java [hunk 1]
```diff
@@ -17,7 +17,7 @@
  */
 public class AllJsTests extends BaseUiTestCase {
 
-    private static final float MIN_COVERAGE_REQUIREMENT = 25;
+    private static final float MIN_COVERAGE_REQUIREMENT = 24;
```
Comment (thenaesh) [inline src/test/java/teammates/test/cases/browsertests/AllJsTests.java]: Why is this done? Perhaps should document it a bit in the commit message.

**Comment by thenaesh** [comment]
@AnaghHegde You may wish to refer to this PR as an example of how commits should be made. Look at the associated diffs for each commit and see how clean and simple they are (and this is a huge PR). Hopefully you can incorporate the lessons learnt in your current and future PRs. It will certainly speed up the review process and let us merge more of your PRs faster. :)

**Comment by AnaghHegde** [comment]
@thenaesh Thank you for the help. I will definitely follow this.

**Comment by HirdayGupta** [comment]
@wkurniawan07 quick question, on which page is the `instructorFeedbacksSpecific.js/.es6` file used? I didn't seem to find it when I went through the entire system looking for the tinyMCE rich text editors to fix #7063.

**Pull request merged**